### PR TITLE
Venmo now supports TFA

### PR DESCRIPTION
**Pull request by cassidoo**
image

### CODE REVIEW

**Comment by Carlgo11** [comment]
So are there any docs on what kind of TFA Venmo support?

**Comment by cassidoo** [comment]
@Carlgo11 Not yet.

**Comment by jdavis** [comment]
@cassidoo Would you be able to add `doc:` and use that blog post until there is more sufficient documentation?

**Comment by mxxcon** [comment]
I think _"we’ll be releasing soon"_ is not the same as _"available right now"_.
I think adding it as `status:` is cool, otherwise I think it's prudent to add this once it's actually released.

**Comment by cassidoo** [comment]
Multi-factor auth _is_ out, just the documentation isn't yet.

**Comment by jdavis** [comment]
Thanks for the clarification, @cassidoo. Would you still be able to add that link like I mentioned [here](https://github.com/jdavis/twofactorauth/pull/1032#issuecomment-86343320)? It is at least a starting point for users that want to read more.

**Comment by cassidoo** [comment]
I'll actually pause until the official post comes out, so we can have better documentation and less PRs.

**Comment by jdavis** [comment]
Okay, we'll wait for you then.

**Comment by ScottSmith95** [comment]
This is now live for Venmo users. They have [some support docs](https://help.venmo.com/customer/portal/articles/1910230-confirm-my-identity-on-sign-in-why-) too. 
img_2808

**Comment by Carlgo11** [comment]
@cassidoo could you add the doc that Mr. Smith provided?

**Comment by cassidoo** [comment]
@Carlgo11 @ScottSmith95 @jdavis We're releasing the "official" announcement this morning!  I'll add those docs. :smile:

**Comment by cassidoo** [comment]
Added docs, here's the official announcement and tweet:

http://blog.venmo.com/hf2t3h4x98p5e13z82pl8j66ngcmry/2015/3/30/updates-to-venmo-security

https://twitter.com/venmo/status/583633322351267841

**Comment by Carlgo11** [comment]
@cassidoo remove `twitter`; and move the `url` up so that's it's just below `name`

**Comment by cassidoo** [comment]
:boom: @Carlgo11

File: _data/payments.yml [hunk 1]
```diff
@@ -82,7 +82,10 @@ websites:
       tfa: No
 
     - name: Venmo
-      img: venmo.png
-      twitter: venmo
-      tfa: No
       url: https://venmo.com
+      twitter: venmo
```
Comment (Carlgo11) [inline _data/payments.yml:86]: uhmmm, you sure? :wink:

Comment (jdavis) [inline _data/payments.yml:86]: I thought we agreed we don't have to be as pedantic with the `twitter:` handle?

Comment (Carlgo11) [inline _data/payments.yml:86]: oh okay. Didn't know about that. never mind then

**Comment by jdavis** [comment]
@cassidoo It should be live.

**Pull request merged**